.azure-pipelines/linux-conda-CI.yml

```diff
@@ -15,24 +15,34 @@ jobs:
     matrix:
       Py37-OnnxGit:
         python.version: '3.7'
-        onnx.version: 'git'
         numpy.version: '==1.16.1'
+        onnx.version: 'git'
         onnxrt.version: '==0.2.1'
+        sklearn.version: '==0.20.3'
       Py36-Onnx141:
         python.version: '3.7'
-        onnx.version: 'git'
         numpy.version: '==1.16.1'
+        onnx.version: '==1.4.1'
         onnxrt.version: '==0.3.0'
+        sklearn.version: '==0.20.3'
       Py36-Onnx130:
         python.version: '3.6'
-        onnx.version: '==1.3.0'
         numpy.version: '==1.16.1'
+        onnx.version: '==1.3.0'
         onnxrt.version: '==0.2.1'
-      Py35-Onnx141:
+        sklearn.version: '==0.20.3'
+      Py35-OnnxGit:
         python.version: '3.5'
+        numpy.version: '==1.16.1'
```

Comment:
Latest numpy is 1.16.2. Shouldn't we be using that instead?

Reply:
probably, it was updated a couple of weeks ago, I don't think this makes a big difference, the big difference is between 0.15 and 0.16.

Reply:
changed